proposals/primary-constructors.md

```diff
@@ -64,7 +64,7 @@ class_body
   
 struct_declaration
   : attributes? struct_modifier* 'partial'? 'record'? 'struct' identifier type_parameter_list?
-    parameter_list? struct_interfaces? type_parameter_constraints_clause* struct_body ';'?
+    parameter_list? struct_interfaces? type_parameter_constraints_clause* struct_body
```

Comment:
i thought we allowed trailing semicolon on thsee types in the language.  or is this only for error recovery?

Reply:
We do, but it's already built into the `class_body` and `struct_body` productions, so also having it here weirdly allows up to two trailing semicolons!